edx/cc2olx
# [BD-27] Convert Common Cartridge links to Static Links in edX

Common Cartridges can contain static files, similar to edX's concept of "Files & Uploads". These files are referenced via the static string $IMS-CC-FILEBASE$ in Common Cartridge XML. During conversion of static HTML pages, these links are not handled, and the links in the courses converted and imported to edX are broken links which don't accurately link to the static files. We fix this behavior by properly converting these links.

**Merge deadline**: None

**Testing instructions**:

1. Checkout this branch
2. Try converting the test course - https://github.com/open-craft/cs7637-knowledge-based-ai-fa20-export
3. Import the course in studio, check out the outline and verify all static files are displayed properly

**Reviewers**
